src/pages/ProgramsPage/WorkshopsPage/JoinWorkshops.jsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useRef } from "react";
+import { useIsVisible } from "../../../components/useIsVisible"; // Adjust the path as necessary
+
+const JoinWorkshops = () => {
+    const ref = useRef();
+    const isVisible = useIsVisible(ref);
+
+    return (
+        <div className="bg-white w-full h-[487.5px] flex items-center justify-center py-72">
+            <div ref={ref} className={`flex flex-col items-center justify-center w-[60rem] h-[30rem] transition-opacity ease-in duration-700 ${isVisible ? "opacity-100" : "opacity-0"}`}>
+                <h1 className="font-karla font-[700] text-5xl text-black pb-3">Boost Your Skills: <span className="text-[#FFA523]">Join Our Workshops</span></h1>
+                <div className="mt-[30px] border-t-4 border-[#FAC80A] w-[10rem]"></div> 
+                <div className="mt-[40px]">
+                    <p className="font-inter text-[16px] leading-8 text-[#3C3C3C    ] text-center">
```

Comment:
I don't think the color is registering on my end, its gray on my side

Reply:
This one I'm not sure what you're seeing, it shows on my end
![image](https://github.com/user-attachments/assets/9231c46a-43d5-4b2b-b9a0-e1f477d2767f)